builtin/k8s/platform.go

```diff
@@ -560,6 +560,7 @@ func configureContainer(
 			Handler:             handler,
 			InitialDelaySeconds: initialDelaySeconds,
 			TimeoutSeconds:      timeoutSeconds,
+			FailureThreshold:    failureThreshold,
```

Comment:
This was a bug - we were getting the default failureThreshold for readiness, which was `3`. This is why Jamie's log shows a 40s timeout instead of 60s like you would expect from our defaults.